Cargo.toml

```
@@ -8,10 +8,10 @@ resolver = "2"
 [workspace.package]
 version = "0.1.0"
 edition = "2021"
-authors = ["Nadir Fejzić"]
+authors = ["Nadir Fejzic <nadirfejzo@gmail.com>", "Kasim Fejzic <kasimfejzo@gmail.com>"]
```

Let's not hide our Bosnian identity here 😛 

```suggestion
authors = ["Nadir Fejzić <nadirfejzo@gmail.com>", "Kasim Fejzić <kasimfejzo@gmail.com>"]
```

---

Please do it in the `pediferrous` crate as well.